### PR TITLE
chore: replaced use of mod https with got

### PR DESCRIPTION
**Pull request by perpk**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (sgkiokas) [inline .vscode/launch.json:1]: Has this file been committed by mistake? Should it be added to `gitignore` in general?

**Review by sgkiokas — COMMENTED** [review]
The updated functionality added looks nice, however, I think we are using a different version of ESLinter caused many other changes in the files (as you can see from the failed pre-check of the PR). 

I have only one comment about a committed file that probably shouldn't be there.

Thanks a lot for the very quick contribution on that issue!

**Comment by sgkiokas** [comment]
Also, if you wouldn't mind, could you please amend the commit message to follow the pattern below?

```
chore/feat/fix: <subject>

<body>

Contributes to/Closes: camelotls/actions-jira-integration#<issue-number>

Signed-off-by: FirstName LastName <email>
```

**Comment by perpk** [comment]
Removed/.gitignored .vscode dir. Fixed formatting violations.

**Review by sgkiokas — APPROVED** [review]
LGTM. Conditionally approving in order to test it in a pipeline as well.

**Comment by sgkiokas** [comment]
@perpk please have a look a the linter once more :)

**Review by sgkiokas — APPROVED** [review]
LGTM

**Pull request merged**